src/pages/error-page.js

```diff
@@ -1,8 +1,69 @@
-import { Typography } from '@mui/material';
+import { Typography } from "@mui/material";
+import { Box } from "@mui/system";
+import { Button } from "@mui/material";
```

Comment:
can we merge these typography and button import like `import {Typography, Button} from '@mui/material';`